Mathlib/Analysis/Normed/Affine/ContinuousAffineMap.lean

```diff
@@ -132,6 +132,29 @@ theorem decomp (f : V →ᴬ[R] W) : (f : V → W) = f.contLinear + Function.con
   rw [coe_mk_const_linear_eq_linear, coe_mk, f.decomp, Pi.add_apply, LinearMap.map_zero, zero_add,
     ← Function.const_def]
 
+/-- The space of continuous affine maps from `P` to `Q` is an affine space over the space of
+continuous affine maps from `P` to `W`. -/
+instance : AddTorsor (P →ᴬ[R] W) (P →ᴬ[R] Q) where
+  vadd f g := { __ := f.toAffineMap +ᵥ g.toAffineMap, cont := f.cont.vadd g.cont }
+  zero_vadd _ := ext fun _ ↦ zero_vadd _ _
+  add_vadd _ _ _ := ext fun _ ↦ add_vadd _ _ _
+  vsub f g := { __ := f.toAffineMap -ᵥ g.toAffineMap, cont := f.cont.vsub g.cont }
+  vsub_vadd' _ _ := ext fun _ ↦ vsub_vadd _ _
+  vadd_vsub' _ _ := ext fun _ ↦ vadd_vsub _ _
+
+@[simp] lemma vadd_apply (f : P →ᴬ[R] W) (g : P →ᴬ[R] Q) (p : P) : (f +ᵥ g) p = f p +ᵥ g p :=
+  rfl
+
+@[simp] lemma vsub_apply (f g : P →ᴬ[R] Q) (p : P) : (f -ᵥ g) p = f p -ᵥ g p :=
+  rfl
+
+lemma vadd_toAffineMap (f : P →ᴬ[R] W) (g : P →ᴬ[R] Q) :
+    (f +ᵥ g).toAffineMap = f.toAffineMap +ᵥ g.toAffineMap :=
+  rfl
+
+lemma vsub_toAffineMap (f g : P →ᴬ[R] Q) : (f -ᵥ g).toAffineMap = f.toAffineMap -ᵥ g.toAffineMap :=
+  rfl
```

Comment:
These look like good `simp` lemmas to me.

Reply:
Made into `simp` lemmas.